dss_datamover/nfs_cluster.py

```diff
@@ -41,7 +41,19 @@ class NFSCluster:
     manager = Manager()
 
     def __init__(self, config={}, user_id="ansible", password="password", logger=None):
-        self.config = config.get("nfs", {})
+        self.config = config.get("fs_config", {}).get("nfs", {})
+        self.nfs_port = config.get("fs_config", {}).get("nfsport", {})
```

Comment:
Default value for nfsport can be set here if not set in config.json or in the cli args